src/WebUI/ClientApp/src/api-authorization/authorize.service.ts

```diff
@@ -144,7 +144,7 @@ export class AuthorizeService {
   public async completeSignOut(url: string): Promise<IAuthenticationResult> {
     await this.ensureUserManagerInitialized();
     try {
-      const response = await this.userManager!.signoutCallback(url);
+      const response = await this.userManager!.signoutRedirectCallback(url);

```

Comment:
[signoutCallback ](https://github.com/authts/oidc-client-ts/blob/main/src/UserManager.ts#L335)has been updated to Promise<void>, so [signoutRedirectCallback](https://github.com/authts/oidc-client-ts/blob/main/src/UserManager.ts#L488) was the closest meaningful and functional option.